scst-store/ingress-multicluster.hbs.md

```diff
@@ -3,16 +3,21 @@
 Supply Chain Security Tools - Store has ingress support by using Contour's HTTPProxy resources. To enable ingress support, a Contour installation must be available in the cluster.
 
 Supply Chain Security Tools - Store's configuration includes two options to configure the proxy: `ingress_enabled` and `ingress_domain`. If needed, you can override the `shared.ingress_domain` Tanzu Application Platform level setting with the `ingress_domain` parameter.
+The store also supports the option to provide a custom certificate under the `tls` option, which needs two options to specify the certificate to use: `secretName` and `namespace`.
+Otherwise, by default, a self-signed certificate will be used.
 
 For example:
 
 ```yaml
 ingress_enabled: "true"
 ingress_domain: "example.com"
 app_service_type: "ClusterIP"  # recommended if ingress is enabled
+tls:  # this section is only needed if a custom certificate is being provided
+  secretName: custom-cert   # name of the custom certificate to use
+  namespace: default        # namespace in which the certificate exists
 ```
 
-Supply Chain Security Tools - Store installation creates an HTTPProxy entry with host routing by using the qualified name `metadata-store.<ingress_domain>` (`metadata-store.example.com`). The create route supports HTTPS communication by using a self-signed certificate with the same subject Alternative Name.
+Supply Chain Security Tools - Store installation creates an HTTPProxy entry with host routing by using the qualified name `metadata-store.<ingress_domain>` (`metadata-store.example.com`). Depending on whether `tls` section is set up in the values yaml, the create route supports HTTPS communication by using a self-signed certificate with the same subject Alternative Name or the provided custom certificate.
```

Comment:
The create route supports HTTPS communication either using the custom cert if the `tls` section is provide, or self-signed certificate with the same subject Alternative Name if the `tls` section is not provided.

Reply:
changed